Integrations/Forescout/Forescout.py

```diff
@@ -0,0 +1,678 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+''' IMPORTS '''
+
+import json
+import requests
+from typing import Dict, List, Tuple, Any, Union
+import xml.etree.ElementTree as ET_PHONE_HOME
+from copy import deepcopy
+from datetime import datetime, timedelta, timezone
+from dateutil.parser import parse as parsedate
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' GLOBALS/PARAMS '''
+
+PARAMS = demisto.params()
+
+CREDENTIALS = PARAMS.get('credentials', {})
+USERNAME = CREDENTIALS.get('identifier', '')
+PASSWORD = CREDENTIALS.get('password', '')
+
+DEX_CREDENTIALS = PARAMS.get('dex_credentials', {})
+DEX_USERNAME = DEX_CREDENTIALS.get('identifier')
+DEX_USERNAME = DEX_USERNAME if DEX_USERNAME != '' else USERNAME
```

Comment:
In the case that the user created the same Username and Password for both the `Web API` and for `Data Exchange (DEX)` as I did, then instead of having to enter both - will just use the same credentials entered for `Web API`